autoblocks/_impl/testing/evaluators/battle.py

```diff
@@ -143,25 +175,13 @@ async def evaluate_test_case(self, test_case: TestCaseType, output: OutputType)
             await self.save_baseline(test_id=test_id, baseline=mapped_output, test_case_hash=test_case.hash())
             return Evaluation(
```

Comment:
we could also return `None` here, which would follow the comment more closely where it says "We save the current challenger as the baseline and skip evaluating"

Reply:
oh ya, i actually had that orginally but switched to this since we didn't have that in js sdk yet. let me switch it back then i'll update js sdk to support none